OPERATOR.rst

```diff
@@ -248,6 +248,22 @@ For an example of how to document failures within a PR `click here`_.
 
 .. _click here: https://github.com/DataBiosphere/azul/pull/3050#issuecomment-840033931
 
+Reindexing a specific catalog in GitLab
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+From the GitLab console, select the reindex job for the pipeline that needs
+reindexing of a targeted catalog. From there, you should see an option for
+defining a key, value pair of variables. These are to be used as an
+environement variable in this run, to enforce any last minute configuration
+changes.
+
+Set ``Key`` to ``azul_current_catalog``, and ``Value`` to the desired catalog to be
+indexed, for example, ``dcp3``. Once you've confirm this is the desired
```

Comment:
```suggestion
Set ``Key`` to ``azul_current_catalog``, and ``Value`` to the desired catalog to
be indexed, for example, ``dcp3``. Once you've confirm this is the desired

```

Reply:
Sorry, my previous suggestion wasn't properly formatted for an 80 character line.